calico-cloud/about/product-comparison.mdx

```diff
@@ -23,78 +24,73 @@ The SaaS version of Calico Enterprise. It adds Image Assurance to scan and detec
-| **Application observability and troubleshooting**            | <center>**Calico Open Source**</center> | <center>**Calico Cloud**</center> | <center>**Calico Enterprise**</center> |
-| Graphical view of deployment (Service Graph)                                |<center>Using third-party tools</center>| <center>&#10003</center> | <center>&#10003</center> |
-| Packet capture                                               |<center>Using third-party tools</center>| <center>&#10003</center> | <center>&#10003</center> |
-| Exportable logs                                              |<center>L3-4 flows, manual integration with any SIEM</center> | Preconfigured Elasticsearch dashboards: flow, audit, bgp, dns, L7 | Preconfigured Elasticsearch dashboards: flow, audit, bgp, dns, L7 |
-| Prometheus for metrics and alert monitoring |<center>Supports manual deployment</center>| <center>&#10003</center> | <center>&#10003</center> |
-| Kibana DNS dashboards                                        |                          | <center>&#10003</center> | <center>&#10003</center> |
-| Traffic Flow Visualizer                                      |                          | <center>&#10003</center> | <center>&#10003</center> |
-| **Kubernetes security posture management** | <center>**Calico Open Source**</center> | <center>**Calico Cloud**</center> | <center>**Calico Enterprise**</center> |
-| Review overall score based on namespace isolation, container image vulnerabilities, and egress access | | <center>&#10003</center> |  |
-| Prioritized list of remediation actions | | <center>&#10003</center> |  |
-| **Cluster mesh**                                             | <center>**Calico Open Source**</center>                     | <center>**Calico Cloud**</center>                            | <center>**Calico Enterprise**</center>                       |
-| Native Kubernetes cross-cluster networking (ToR and IP-in-IP) | <center>&#10003</center> | <center>&#10003</center> | <center>&#10003</center> |
-| Multi-cluster management with RBAC integration | <center>CLI only</center> |  | <center>GUI and CLI</center> |
-| Federated cross-cluster networking (VXLAN) |  | <center>&#10003</center> | <center>&#10003</center> |
-| Federated identity-aware policy and services enforcement |  | <center>&#10003</center> | <center>&#10003</center> |
-| **Threat defense**                                           | <center>**Calico Open Source**</center> | <center>**Calico Cloud**</center> | <center>**Calico Enterprise**</center> |
-| Container threat detection                                   |                          | <center>&#10003</center> |                          |
-| Workload-centric Web Application Firewall (WAF)              |                          | <center>&#10003</center> | <center>&#10003</center> |
-| Honeypods to see intruder activity                           |                          | <center>&#10003</center> | <center>&#10003</center> |
```

Comment:
Changing this file seems to have reformatted the whole file, guessing some markdown plugin in the IDE / text editor. Not sure if that was intentional or has any side effects.